oe_theme.theme

```diff
@@ -734,6 +734,17 @@ function _oe_theme_preprocess_pattern_social_media_links(array &$variables): voi
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_featured_item(array &$variables): void {
+  $title = [
+    'label' => $variables['title'],
+    'path' => $variables['link']['path'],
+  ];
+  $variables['title'] = $title;
```

Comment:
This should happen in the twig templates, not in pattern preprocess, since more specific preprocesses are ran after this one setting variable here might create a problem.